ushahidi/platform-client
# Language and intercom features

This pull request makes the following changes:
- Adding language settings
- Adding intercom feature

Test these changes by:
- Ensure languages are available from settings menu
- Check that language automatically updates when settings saved

Fixes ushahidi/platform# .

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/126)
